homeassistant/components/bond/cover.py

```diff
@@ -63,10 +61,26 @@ def device_info(self) -> Optional[Dict[str, Any]]:
         """Get a an HA device representing this cover."""
         return {ATTR_NAME: self.name, "identifiers": {(DOMAIN, self._device.device_id)}}
 
+    @property
+    def assumed_state(self) -> bool:
+        """Let HA know this entity relies on an assumed state tracked by Bond."""
+        return True
+
+    @property
+    def should_poll(self) -> bool:
+        """Let HA know it should periodically poll this entity for state changes."""
+        return True
+
+    def update(self):
+        """Fetch assumed state of the cover from the hub using API."""
+        state: dict = self._bond.getDeviceState(self._device.device_id)
+        cover_open = state.get("open", None)
```

Comment:
`None` is the default value returned if the key is missing in the dict.

Reply:
Thanks, cleaned up.